src/rebase.c

```diff
@@ -967,6 +967,13 @@ static int rebase_commit__create(
 		goto done;
 	}
 
+	if ((!message || strcmp(message, git_commit_message(current_commit)) == 0) &&
+	    (!message_encoding || strcmp(message_encoding, git_commit_message_encoding(current_commit)) == 0) &&
```

Comment:
Well, committer is a non-optional argument to this function, for one.

Also, what would happen if a user passes author or comitter - would we have to check whether they differ from the previous atuhor/committer? If that makes sense, I can implement the necessary changes.